src/Files.App/Views/Properties/MainPage.xaml

```diff
@@ -1,11 +1,11 @@
 ﻿<Page
-	x:Class="Files.App.Views.Properties"
+	x:Class="Files.App.Views.Properties.MainPage"
```

Comment:
I don't think `MainPage` is a good, should be something unique to properties.

Reply:
MainPropertiesPage?

Reply:
👍